pkgs/applications/networking/instant-messengers/fractal/default.nix

```diff
@@ -92,6 +92,10 @@ stdenv.mkDerivation rec {
     patchShebangs scripts/meson_post_install.py scripts/test.sh
   '';
 
+  preConfigure = ''
+    export GETTEXT_DIR="${gettext}/lib"
```

Comment:
Looking at https://github.com/danigm/gettext-rs/blob/61938b9f5f1d3bdc31f9839f53fabe5ccf136a78/gettext-sys/build.rs#L89-L115, this will produce incorrect directories.

Possibly it works by accident?

Reply:
Setting `GETTEXT_DIR` to `${gettext}` now.